azurerm/resource_arm_netapp_account_test.go

```diff
@@ -0,0 +1,224 @@
+package azurerm
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/tf"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func TestAccAzureRMNetAppAccount(t *testing.T) {
+	// NOTE: this is a combined test rather than separate split out tests since
+	// Azure allows only one active directory can be joined to a single subscription at a time for NetApp Account.
+	// The CI system runs all tests in parallel, so the tests need to be changed to run one at a time.
+	testCases := map[string]map[string]func(t *testing.T){
+		"Resource": {
+			"basic":    testAccAzureRMNetAppAccount_basic,
+			"complete": testAccAzureRMNetAppAccount_complete,
```

Comment:
we also need to add a Requires Import test here - example: https://github.com/terraform-providers/terraform-provider-azurerm/blob/master/azurerm/resource_arm_resource_group_test.go#L38-L64

Reply:
Fixed